config/sync/views.view.stanford_event_series.yml

```diff
@@ -408,7 +408,7 @@ display:
             offset: false
             offset_label: Offset
           views_infinite_scroll:
-            button_text: 'Load More'
+            button_text: 'Load More Events'
```

Comment:
I thought it was desired to have "View More Events". But I might have missed something.

Reply:
Per the ticket, it should be `Load More [whatever thing]` but I'm happy to make it whatever we agree on.

Reply:
Let's check with Ishi on it real quick. I don't see her input on the ticket. I'm good either way, I'd just rather get it done once.